docs/core/configuration.md

```diff
@@ -404,6 +438,7 @@ create_empty_blocks_interval = "0s"
 
 # Reactor sleep duration parameters
 peer_gossip_sleep_duration = "100ms"
+peer_gossip_intraloop_sleep_duration = "0s"
```

Comment:
Why is this change here?

Reply:
When I added the sample config file to the docs, all I did was `cometbft init` locally and copy/paste the contents of `~/.cometbft/config/config.toml` into this doc. The fact that this was missing implies the doc was already out of date on `main`.

Reply:
Ok good, I was just double checking that some testing config did not make it into the PR. Sounds good!